### PR TITLE
Update RELEASES.md

### PR DESCRIPTION
**Pull request by jtwinrelevanceai**

### CODE REVIEW

**Comment by boba-and-beer** [comment]
@ofrighil do these make sense to you?

File: RELEASES.md [hunk 1]
```diff
@@ -19,21 +19,24 @@ If they have been manually tested, proceed to do the following:
...
-- Create a new branch from development for the version you are released following SemVer2.0 formatting, for example: `v0.28.0` or `v0.53.2`, this will automatically trigger `readthedocs` to create documentation once the new branch has been pushed.
-- Submit a Pull Request for the new version branch (`v0.28.0`) into `main`
-- Wait for tests to finish 
+- Go to https://relevanceai.readthedocs.io/en/latest/
+- Click on Read the Docs in the bottom left corner, and check to make sure the new version is listed.
 
 ### Step 4: Releasing `Relevance AI` package
-- Merge this PR, this will trigger a Github action that will update the package.
-- You will also want to do a Release on Github for fun.
+- By this time, the tests will be close to down
```
Comment (ofrighil) [inline RELEASES.md:38]: should this be "... close to done"?

**Review by ofrighil — COMMENTED** [review]
Yes it makes sense, just had one comment.

File: RELEASES.md (hunk 1 above)
Comment (boba-and-beer) [reply RELEASES.md:38]: Oh yep - will fix

**Pull request merged**